Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -1825,7 +1825,7 @@ static NSInteger _colCompare(id block1, id block2, void *)
         Element* blockElement = _blockLevelElementForNode(element.parentInComposedTree());
         NSString *breakClass = element.getAttribute(classAttr);
         NSString *blockTag = blockElement ? (NSString *)blockElement->tagName() : nil;
-        BOOL isExtraBreak = [@"Apple-interchange-newline" isEqualToString:breakClass];
+        BOOL isExtraBreak = [AppleInterchangeNewline.createNSString() isEqualToString:breakClass];
```

Comment:
Is this a very hot codepath? I wonder if the intermediate RetainPtr creation/destruction just for the comparison is an unnecessary perf cost.